.gitignore

```diff
@@ -39,3 +39,9 @@ erl_crash.dump
 /.elixir_ls/
 *~
 *.orig
+
+*.iml___jb_old___
+
+*.iml
+
+.idea/
```

Comment:
This is just for people who use IntelliJ to work on RIG (I tried writing Elixir in IntelliJ. It works. It wasn't for me but I thought that if someone does like it, having the IntelliJ project files in our gitignore doesn't harm anyone).